npm/grep/test.spec.js

```diff
@@ -0,0 +1,288 @@
+const Fixtures = require('@tooling/system-tests')
+const FixturesScaffold = require('@tooling/system-tests/lib/dep-installer')
+const execa = require('execa')
+const path = require('path')
+const fs = require('fs-extra')
+
+const scaffoldGrepProject = async () => {
+  const project = 'grep'
+  const projectPath = Fixtures.projectPath(project)
+
+  Fixtures.removeProject(project)
+  await Fixtures.scaffoldProject(project)
+  await FixturesScaffold.scaffoldProjectNodeModules({ project })
+
+  return projectPath
+}
+
+const runCommandInProject = async (command, projectPath, _args) => {
+  const [ex, ...args] = command.split(' ')
+
+  // _args is used for scenarios where we are filtering with spaces
+  return await execa(ex, _args ? _args : args, { cwd: projectPath, stdio: 'inherit' })
+}
+
+const cypressGrepPackagePath = path.join(__dirname)
+
+describe('@cypress/grep', () => {
+  let projectPath
+
+  before(async () => {
+    projectPath = await scaffoldGrepProject(cypressGrepPackagePath)
+
+    await runCommandInProject(`yarn add @cypress/grep@file:${cypressGrepPackagePath}`, projectPath)
+  })
+
+  it('run Unit Tests', async () => {
+    await runCommandInProject('npx cypress run --spec cypress/e2e/unit.js --config specPattern="**/unit.js"', projectPath)
+  })
```

Comment:
there should not be unit tests in system-tests. These should live in `npm/grep`

Reply:
Resolved